lib/github.rb

```diff
@@ -132,16 +151,28 @@ def self.puts_pull_request(pr, options = {})
 
     pr["reviews"].each do |user, state|
       if state == "APPROVED"
-        puts_with_prefix.call " \e[92m\e[1m✔ \e[0m #{user}"
+        if options[:color]
+          puts_with_prefix.call " \e[92m\e[1m✔ \e[0m #{user}"
+        else
+          puts_with_prefix.call " ✔  #{user}"
+        end
       elsif state == "CHANGES_REQUESTED"
-        puts_with_prefix.call " \e[91m\e[1m±\e[0m  #{user}"
+        if options[:color]
+          puts_with_prefix.call " \e[91m\e[1m±\e[0m  #{user}"
+        else
+          puts_with_prefix.call " ±  #{user}"
+        end
       elsif state == "COMMENTED"
-        puts_with_prefix.call "💬  #{user}"
+        puts_with_prefix.call " 💬  #{user}"
```

Comment:
this is just an extra space for consistency with the other cases, not sure if this was deliberately different tho

Reply:
It is deliberate, the character is the width of two normal characters (at least in my terminal with my font), so this change will cause misalignment